docs/app/interfaces/Migration.ts

```diff
@@ -0,0 +1,5 @@
+export interface Migration {
+  title: string;
+  href: string;
```

Comment:
This isn't the full href, so probably better to give it a more accurate name like `storybookPath`.